main.cpp

```diff
@@ -23,6 +23,7 @@
 
 #define CMD_WRITE          0x02
 #define CMD_READ           0x03
+#define CMD_FASTREAD       0x0B
```

Comment:
```suggestion
#if TARGET_NORDIC  // target specific check 
#define CMD_READ       0x0B     // Fast Read
#else
#define CMD_READ           0x03
#endif
```

Reply:
Fast read (0x0B) is more common with newer platforms/flash parts, especially ones that support QSPI. With that said, it makes sense to check the target and set the read command based on that. Thank you for the suggestion.